### DISCUSSION TITLE
Possible solution for CROWS turrets dropping a lot of ammo

### DISCUSSION THREAD

**Discussion by Saicchi** [CleverRaven/Cataclysm-DDA#49042] [Ideas]
CROWS turrets drop a lot of ammo. Just killing a single turret nets the survivor ~300 .50 BMG bullets, ~1000 .308 bullets or ~1600 .223 bullets.

With a bit of experience these turrets are easy to kill, and the amount of ammo they give trivializes the game, as the player has a high amount of ammo and currency to use when trading.

I suggest keeping the amount they drop, but changing it so they're turret specific and won't fit in a normal weapon the player can have.

Instead the player should disassemble them get their primer, and gunpowder to craft more bullets. And smelting the casing should give back some materials for the craft of more ammo.

That won't stop a player with a full setup for reloading ammo from getting high amounts of it, provided they have the materials for it.

**Comment by Raikiri** [comment]
I don't think the devs see it as a problem, at least as far as I remember from previous discussions. The rationale is that according to wikipedia, real-life CROWS II is indeed typically equipped with 1000+ rounds of ammo.

If it was considered to be a problem, there are lots of ways it could have been addressed already.

**Comment by anothersimulacrum** [comment]
Additional context: https://github.com/CleverRaven/Cataclysm-DDA/pull/49145

**Comment by krulunio** [comment]
I think turrets should spawn with much less ammo and many casings and corpses around them. Also, from my experience, lab turrets are easiest to deal with, so maybe we should change them into laser turrets?

**Reply by Saicchi** [reply]
Not viable. Read the post linked by anothersimulacrum above.
